src/QueryType/ManagedResources/RequestBuilder/Synonyms.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Solarium\QueryType\ManagedResources\RequestBuilder;
+
+use RuntimeException;
+use Solarium\Core\Client\Request;
+use Solarium\Core\Query\AbstractRequestBuilder as BaseRequestBuilder;
+use Solarium\Core\Query\QueryInterface;
+use Solarium\QueryType\ManagedResources\Query\Synonyms as SynonymsQuery;
+use Solarium\QueryType\ManagedResources\Query\Synonyms\Command\AbstractCommand;
+
+
+class Synonyms extends BaseRequestBuilder {
```

Comment:
This request builder is very similar to Stopwords. I would avoid that redundant code using a base class or a trait.